frontend/server/src/Controllers/Contest.php

```diff
@@ -704,10 +704,15 @@ public static function apiOpen(\OmegaUp\Request $r) {
         [
             'needsBasicInformation' => $needsInformation,
             'requestsUserInformation' => $requestsUserInformation
-        ] = \OmegaUp\DAO\Contests::getNeedsInformation($response['contest']->problemset_id);
+        ] = \OmegaUp\DAO\Contests::getNeedsInformation(
+            $response['contest']->problemset_id
+        );
 
-        if ($needsInformation && (!$r->identity->country_id || !$r->identity->state_id
-            || !$r->identity->school_id)
+        if (
+            $needsInformation
+              && (!$r->identity->country_id
+              || !$r->identity->state_id
+                || !$r->identity->school_id)
```

Comment:
```suggestion
              && (!$r->identity->country_id
                  || !$r->identity->state_id
                  || !$r->identity->school_id)
```

así va a estar bien alineado.